package.json

```diff
@@ -33,6 +33,7 @@
     "libxmljs": "^0.18.7",
     "lodash": "^4.17.4",
     "mime-types": "^2.1.14",
+    "request-promise-native": "^1.0.4",
```

Comment:
does `request` not need to be installed as well? I think it's a peer dependency. It mentions at the top of the docs: https://www.npmjs.com/package/request-promise-native